scripts/operations_tests/operations_tests.py

```diff
@@ -222,30 +239,43 @@ def load_post_operation_image_stack(filepath):
 
 
 def load_image_stack():
-    filename_group = FilenameGroup.from_file(config_manager.load_sample)
-    filename_group.find_all_files()
-    image_stack = loader.load(filename_group=filename_group)
-    return image_stack
+    try:
+        load_sample_path = config_manager.load_sample
+        filename_group = FilenameGroup.from_file(load_sample_path)
+        filename_group.find_all_files()
+        image_stack = loader.load(filename_group=filename_group)
+        return image_stack
+    except FileNotFoundError as err:
+        error_msg = f"File not found at path: {load_sample_path}"
+        logging.error(error_msg)
+        raise FileNotFoundError(error_msg) from err
+    except Exception as err:
+        error_msg = f"Error while loading image stack: {err}."
+        logging.error(error_msg)
+        raise Exception(error_msg) from err
```

Comment:
No need to explicitly catch generic exceptions. We'll get the stack trace if something happens.